backend/src/firmware/boards-infos.json

```
@@ -96,6 +162,17 @@
                 "offset": 65536
             }
         ],
-        "board": "esp32dev"
+        "board": "esp32dev",
+        "defaults": {
+            "imuSDA": "2",
+            "imuSCL": "0",
+            "imuInts": ["255", "255"],
+            "led": "255",
+            "enableLed": false,
+            "ledInverted": false,
+            "batteryPin": "A0",
```

```suggestion
            "batteryPin": "255",
```
https://github.com/SlimeVR/SlimeVR-Tracker-ESP/blob/main/src/defines.h#L136